recipes/bzip2/all/conanfile.py

```diff
@@ -100,13 +102,8 @@ def _module_file(self):
         return "conan-official-{}-variables.cmake".format(self.name)
 
     def package_info(self):
-        self.cpp_info.names["cmake_find_package"] = "BZip2"
-        self.cpp_info.names["cmake_find_package_multi"] = "BZip2"
         self.cpp_info.builddirs.append(self._module_subfolder)
         self.cpp_info.build_modules["cmake_find_package"] = [os.path.join(self._module_subfolder, self._module_file)]
         self.cpp_info.libs = ["bz2"]
-
-        if self.options.build_executable:
-            bin_path = os.path.join(self.package_folder, "bin")
```

Comment:
Got appending the path implicit somehow? If yes, through what?

Reply:
We expect the consumers to use the `VirtualRunEnv` when need it instead of filling the PATH by default with stuff with might never need.